tests/www/employees_views/test_detail.py

```diff
@@ -211,3 +211,11 @@ def test_link_immersion_facile(self, client, snapshot):
         assert response.context["link_immersion_facile"] == immersion_search_url(approval.user)
         alert = parse_response_to_soup(response, selector="#immersion-facile-opportunity-alert")
         assert str(alert) == snapshot(name="alerte à l'opportunité immersion facile PASS expirant dans longtemps")
+
+        approval.start_at = today + datetime.timedelta(days=1)
```

Comment:
Évite une erreur vers minuit.
```suggestion
        approval.start_at = today + datetime.timedelta(days=2)
```